sites/public/__tests__/components/listing/GetApplication.test.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react"
+import { render, cleanup } from "@testing-library/react"
+import { GetApplication } from "../../../src/components/listing/GetApplication"
+
+afterEach(cleanup)
+
+describe("<Applications>", () => {
+  it("renders with all optional fields", () => {
+    const { getByText, getAllByText } = render(
+      <GetApplication
+        onlineApplicationURL={"online-app-url"}
+        applicationsOpen={true}
+        applicationsOpenDate={"November 20th, 2021"}
+        paperApplications={[
+          { fileURL: "file-url-en", languageString: "English" },
+          { fileURL: "file-url-es", languageString: "Spanish" },
+        ]}
+        paperMethod={true}
+        postmarkedApplicationsReceivedByDate={"November 30th, 2021"}
+        applicationPickUpAddressOfficeHours={"M-F 9am-5pm"}
+        applicationPickUpAddress={{
+          city: "City",
+          state: "State",
+          street2: "Street 2",
+          street: "Pick Up Address Street",
+          zipCode: "90210",
+        }}
+        preview={false}
+      />
+    )
+    expect(getByText("How to Apply")).toBeTruthy()
+    expect(getByText("Apply Online")).toBeTruthy()
+    expect(getByText("Apply Online").closest("a")?.getAttribute("href")).toBe("online-app-url")
```

Comment:
I know these are existing tests and what is here works so no need to change. But to do a more modern approach of tackling this you could do
`expect(getByRole("link", { name: "Apply Online" })).toHaveAttribute("href", "online-app-url")`.

Also all of the `toBeTruthy()` can be `toBeInTheDocument()`